### ISSUE TITLE
Missing method detected by PHP 7.4 in PhpSpec\Loader\StreamWrapper

### ISSUE THREAD

**Issue by pfazzi** [phpspec/phpspec#1274]
When I exec phpspec with PHP 7.4.0RC4 i get the following warnig:
```bash
PHP Warning:  require_once(): PhpSpec\Loader\StreamWrapper::stream_set_option is not implemented! in /projects/PED/NutriBees/vendor/phpspec/phpspec/src/PhpSpec/Loader/ResourceLoader.php on line 56
```

**Comment by ddziaduch** [comment]
This is already fixed in the `master` by @ciaranmcnulty

**Comment by ciaranmcnulty** [comment]
Yep, released soon

**Comment by ibolee** [comment]
@ciaranmcnulty  @ddziaduch 
hello,engineers

Hello, I also have this problem, could you please tell me where the solution link is?

best regards！

**Comment by ciaranmcnulty** [comment]
This is fixed in version 6.1.0 or later
